hoodie-common/src/main/java/com/uber/hoodie/config/AbstractCommandConfig.java

```diff
@@ -0,0 +1,123 @@
+/*
```

Comment:
Didn't realize this is still in hoodie-common. Can you move this and InvalidCommandConfigException to hoodie-utilities. Only CLI related configs can inherit from this class. to keep it simple. This would mean that HiveSyncConfig changes can be reverted